tests/unit/admin/test-class-wc-payments-admin.php

```diff
@@ -91,9 +96,7 @@ public function test_it_renders_payments_badge_if_stripe_is_not_connected() {
 		// The badge is temporarily not shown at all. See
 		// class-wc-payments-admin.php line 227. This should be removed when
 		// implementing https://github.com/automattic/woocommerce-payments/issues/2071.
-		if ( true === true ) {
-			return;
-		}
+		static::markTestSkipped( 'Not yet implemented.' );
```

Comment:
Unrelated change. Previously, this test was showing as "risky" during test runs. This feels a little most explicit.